src/include/firewall/uifunctions.rb

```diff
@@ -135,21 +144,33 @@ def SaveAndRestart
     def StartNow
       UI.OpenDialog(Label(_("Starting firewall...")))
       SuSEFirewall.SetStartService(true)
-      ret = SuSEFirewall.StartServices
+      success = SuSEFirewall.StartServices
+      Popup.Error(
+        _(
+          "The firewall could not be started.\n" +
+          "Please verify your system and try again."
+        )
+      ) unless success
       UI.CloseDialog
 
-      ret
+      success
     end
 
     # Function stops Firewall services and sets firewall
     # to be stopped after exiting YaST
     def StopNow
       UI.OpenDialog(Label(_("Stopping firewall...")))
       SuSEFirewall.SetStartService(false)
-      ret = SuSEFirewall.StopServices
+      success = SuSEFirewall.StopServices
+      Popup.Error(
+        _(
+          "The firewall could not be stopped.\n" +
+          "Please verify your system and try again."
+        )
```

Comment:
yeah, but consider that also enable failed, disable failed, restart failed, reload failed or something else. I worry it can be a bit mess later.